### ISSUE TITLE
Update IL algorithms to include provider closures 

### ISSUE THREAD

**Issue by csprayregen** [pieforproviders/pieforproviders#413]
## ✨ What new feature do you want to see?
How do state subsidy billing rules handle providers' closures, e.g. for holiday or vacation? Do we need to update our algorithms to account for this difference?

**Comment by jacqzwy** [comment]
Would the eligible days uploaded for the providers already include this adjustment for closures?

If not, we would need more information than what we currently ask providers to do this adjustment on our end. When eligible days are determined, they are determined according to the client's work schedules. (See 
[establishing_eligibility.pdf](https://github.com/pieforproviders/pieforproviders/files/5404238/establishing_eligibility.pdf)
and 
[provider_closures.pdf](https://github.com/pieforproviders/pieforproviders/files/5404241/provider_closures.pdf)
).

Thus, in order to adjust for closures we would need to know
1. What days the provider is closed
2. For each child, if the eligible day based on the client work schedule falls on a day a provider is closed

**Comment by csprayregen** [comment]
Yeah, this is super confusing @jacqzwy ! I'm not seeing an easy way to integrate this into our current workflow. I'm going to move this back to the backlog column for the moment. Let's de-prioritize and start piloting this in IL, and revisit when we have capacity and/or if we see that this is a relevant issue coming up in our pilot.

**Comment by csprayregen** [comment]
This is closely related to a set of questions regarding provider hours of operation that are asked but not yet answered here: https://docs.google.com/document/d/1yHGt95L1mP_94mi1pGyLKaaL-YSiW-8xvckTpv9WDGs/edit#bookmark=id.t3uaw6jvh7sw

Will move this to backlog and a later milestone while we work thru partners to find answers to these key questions.
